src/utils.js

```diff
@@ -219,7 +219,7 @@ function setSelectionRangeIfNecessary(
 }
 
 const FOCUSABLE_SELECTOR = [
-  'input:not([disabled])',
+  'input:not([type=hidden]):not([disabled])',
```

Comment:
Actually, the hidden attribute could be applied to any element. I think we need to add this to all of the selectors here.

Reply:
I just checked in Chrome: `<div tabIndex="0" type="hidden">` has no effect.
I don't think it is specified to have any meaning on other elements.

Reply:
Oh, duh. I was thinking about the `hidden` attribute itself (like `<button hidden="true">`).

But yeah, the `hidden` type is what we're dealing with here and that works fine. Thanks!

I wonder whether we should/need to fix how we handle the hidden attribute though 🤔

I'll merge this as-is. Thanks!

Reply:
@lazytype, I mean, I'm happy to merge after you sort out @ph-fritsche's comment https://github.com/testing-library/user-event/pull/467/files#r504487490